index.js

```diff
@@ -187,9 +218,15 @@ PersonaClient.prototype.getPublicKey = function getPublicKey(cb, refresh) {
  * Validate bearer token locally, via JWT verification.
  * @param {string} token -  Token to validate.
  * @param {string=} scope - Optional requested scope that if provided, is also used to validate against.
+ * @param {string=} xRequestId - Optional request ID to pass through.
  * @callback next - Called with either an error as the first param or "ok" as the result in the second param.
  */
-PersonaClient.prototype.validateToken = function (token, scope, next) {
+PersonaClient.prototype.validateToken = function (token, scope, xRequestId, next) {
+    if (_.isFunction(xRequestId)) {
+        next = xRequestId; // third param is actually next(), for backwards compat.
```

Comment:
Can we compromise and add a deprecation warning log? I'm just weary of future code maintenance in this area.